oat-sa/extension-tao-itemqti
# chore: bundle assets

Related to: https://oat-sa.atlassian.net/browse/AUT-2591

The bundles generated for the release [29.6.5](https://github.com/oat-sa/extension-tao-itemqti/releases/tag/v29.6.5) contain issues. Most probably they were generated with an outdated version of the TAO bundler.

This PR brings fixed bundles.

**How to test:**
- checkout this fix
- make sure the `DEBUG_MODE` is disabled in `config/generis.conf/php`
- log in to the back-office
- go to the Items
- select an existing item, or create one
- try to add interactions

Note: the console should not issue critical errors, and the UI should work seamlessly.
